frontend_app_file/src/css/index.styl

```diff
@@ -1,3 +1,4 @@
+@import "~tracim_frontend_lib/src/css/Generic.styl"
```

Comment:
Just a question: it's necessary to import Generic all times? if you import just at Tracim.jsx it doesn't works?

Reply:
I assumed it was since `Variable.styl` is imported everywhere too, I'll try without.